library/src/main/java/org/jellyfin/apiclient/model/entities/PersonType.java

```diff
@@ -3,38 +3,7 @@
 /**
  Struct PersonType
  */
-public class PersonType
+public enum PersonType
 {
-    /**
-     The actor
-     */
-    public static final String Actor = "Actor";
-    /**
-     The director
-     */
-    public static final String Director = "Director";
-    /**
-     The composer
-     */
-    public static final String Composer = "Composer";
-    /**
-     The writer
-     */
-    public static final String Writer = "Writer";
-    /**
-     The guest star
-     */
-    public static final String GuestStar = "GuestStar";
-    /**
-     The producer
-     */
-    public static final String Producer = "Producer";
-    /**
-     The conductor
-     */
-    public static final String Conductor = "Conductor";
-    /**
-     The lyricist
-     */
-    public static final String Lyricist = "Lyricist";
+Actor, Director, Composer, Writer, GuestStar, Producer, Conductor, Lyricist, Other
```

Comment:
Formatting looks a bit off. I think a tab (or 4 spaces) is missing at the start of the row.
If you use IntelliJ or Android Studio just use the built-in formatter

Reply:
Whoops, fixing. I also deleted the nondescriptive comment at the file start now as well.